pipeline_tools/tests/shared/submission/test_create_analysis_file.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
this test is invalid, you actually need to give it a real path to a real metadata file